jena-ontapi/src/main/resources/META-INF/services/org.apache.jena.sys.JenaSubsystemLifecycle

```diff
@@ -0,0 +1,3 @@
+org.apache.jena.riot.system.InitRIOT
+org.apache.jena.sparql.system.InitARQ
+org.apache.jena.rdfs.sys.InitRDFS
```

Comment:
These should not be here. They are in other ServiceLoader files and wil be run from there. That might be the initialization problem. Putting them here makes them get called twice.

Here, give a class in this module:
```
org.apache.jena.ontapi.sys.InitOntAPI
```
then new package and class:

org.apache.jena.ontapi.sys.InitOntAPI

```
public class InitOntAPI implements JenaSubsystemLifecycle {

    @Override
    public void start() {
         // Any initialization for this module.  This may be nothing.
    }

    @Override
    public void stop() {}
    
    @Override
    public int level() {
        return 15 ;                         -- this is between jena-core and jena-arq RIOT.
    }
}
```